frontendExternalSystem/src/main/java/com/virtuslab/gitmachete/frontend/externalSystem/MacheteIconProvider.java

```diff
@@ -0,0 +1,15 @@
+package com.virtuslab.gitmachete.frontend.externalSystem;
+
+// "externalIconProvider" is available since 2020.1. The following lines shall be uncommented after 2019.3 support is dropped.
```

Comment:
Pls open an task umbrella issue on GH for the migrations/improvements to do once 2019.3 is dropped + put the issue number in `TODO (#...)` here

Reply:
Also, pls leave a comment what's the current 2019.3-compatible workaround

Reply:
issue #339 created